File: OP/2Semester/Lab2/Lab2(py)/Lab2(py)/ProgramTV.py
import datetime

class ProgramTV(object):
    def __init__(self):
        self.title=''
        self.startTime=datetime.datetime.now()
        self.endTime=datetime.datetime.now()
        self.timeSpan=self.endTime-self.startTime

    def Print(self):
        print(f"Title: {self.title}  Start: {self.startTime.time()}  End: {self.endTime.time()}  Lasts: {self.timeSpan}")
   

File: Programming Foundations/2semester/Lab2/Lab2/ProgramTV.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab2
{
    
    class ProgramTV
    {
        public DateTime StartTime { get; set; }
        public DateTime EndTime { get; set; }
        public TimeSpan TimeSpan { get; set; }
        public string Title { get; set; }
        public ProgramTV(DateTime start, DateTime end, string title)
        {
            StartTime = start;
            EndTime = end;
            Title = title;
            TimeSpan = new TimeSpan(EndTime.Hour-StartTime.Hour, EndTime.Minute-StartTime.Minute, EndTime.Second - StartTime.Second);
        }
        public ProgramTV() { }
        public void Print()
        {
            Console.WriteLine($"Title: {Title}  Start: {StartTime.TimeOfDay}  End: {EndTime.TimeOfDay}  Lasts: {TimeSpan}");
        }
    }
}
